package.json

```diff
@@ -27,16 +27,16 @@
   "devDependencies": {
     "envsub": "^3.1.0",
     "nwb": "0.23.x",
-    "react": "^16.5.2",
-    "react-dom": "^16.5.2"
+    "react": "^16.8.0",
```

Comment:
I'll probably use this update to narrow the peerDependencies and prepare for react 17
` "react": "^16.8.0 || ^17.0.0"`

Reply:
Agreed and done in https://github.com/didomi/react/pull/25/commits/ddc1601a18512c1663d9e1689affe2d7c40f2ae8